clients/geth/specular/sbin/export_genesis.sh

```diff
@@ -5,9 +5,14 @@ SBIN="`cd "$SBIN"; pwd`"
 . $SBIN/configure_system.sh
 cd $DATA_DIR
 
+mkdir tmp_data/
+
 args=(
-    --datadir ./data_sequencer
+    --datadir ./tmp_data/
     --verbosity 0
 )
-$GETH_SPECULAR_DIR/build/bin/geth init --datadir ./data_sequencer ./base_genesis.json
+
+$GETH_SPECULAR_DIR/build/bin/geth init --datadir ./tmp_data ./genesis.json
 $GETH_SPECULAR_DIR/build/bin/geth "${args[@]}" dump 0
+
+rm -r tmp_data/
```

Comment:
`-rf` instead?

Reply:
i think it might be nice not to suppress errors for easier debugging. But I don't have strong opinions, any specific reason you suggested it, or just general preference?

Reply:
yeah I think you're right